.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+### Milestone (Odoo version)
```

Comment:
For what I have seen in Odoo, commit message gets injected somewhere in the template. You can just delete all the template if you still prefer old behavior.

Another option is to only put template for issues.